samples/ttam/src/main/java/com/google/api/services/samples/ttam/SimpleExample.java

```diff
@@ -0,0 +1,127 @@
+        this.clientSecret = clientSecret;
+        this.httpTransport = new NetHttpTransport();
+        this.jsonFactory = new JacksonFactory();
+    }
+
+    public static void main(String[] args) throws Throwable {
+        File directory = new File(System.getProperty("user.home"), ".store/ttam");
+        if (args.length != 2) {
+            throw new AssertionError("client_id and client_secret are required");
+        }
+        SimpleExample simpleExample = new SimpleExample(new FileDataStoreFactory(directory), args[0], args[1]);
+        Genotype genotype = simpleExample.retrieveGenotype();
+        System.out.println("Genotype @ i3003137, a SNP associated with Sickle Cell Anemia: " + genotype.i3003137);
+    }
+
+    public Genotype retrieveGenotype() throws Exception {
+        final Credential credential = authorize();
+
+        HttpRequestInitializer initializer = new HttpRequestInitializer() {
+            public void initialize(HttpRequest request) throws IOException {
+                credential.initialize(request);
+                request.setParser(new JsonObjectParser(jsonFactory));
+            }
+        };
+        HttpRequestFactory requestFactory = httpTransport.createRequestFactory(initializer);
+
+        GenericUrl userUrl = new GenericUrl(BASE_URL + "/1/user/");
+        HttpRequest userRequest = requestFactory.buildGetRequest(userUrl);
+        HttpResponse userResponse = userRequest.execute();
+        Account account = userResponse.parseAs(Account.class);
+        Profile profile = account.profiles.get(0);
+
+        GenericUrl genotypesUrl = new GenericUrl(BASE_URL + "1/genotypes/" + profile.id + "/?locations=i3003137");
+        HttpRequest genotypesRequest = requestFactory.buildGetRequest(genotypesUrl);
+        HttpResponse genotypesResponse = genotypesRequest.execute();
+        return genotypesResponse.parseAs(Genotype.class);
+    }
+
+    private Credential authorize() throws Exception {
+        HttpExecuteInterceptor credentials = new ClientParametersAuthentication(this.clientId, this.clientSecret);
+        AuthorizationCodeFlow flow = new AuthorizationCodeFlow.Builder(authorizationHeaderAccessMethod(),
+                                                             httpTransport,
+                                                             jsonFactory,
+                                                             new GenericUrl(TOKEN_SERVER_URL),
+                                                             credentials,
+                                                             this.clientId,
+                                                             AUTHORIZATION_SERVER_URL)
+                                                            .setScopes(SCOPES)
+                                                            .setDataStoreFactory(dataStoreFactory).build();
+        VerificationCodeReceiver receiver = new LocalServerReceiver.Builder().setHost(DOMAIN)
```

Comment:
`.setHost()` should be on it's own line.  Don't think DOMAIN is the right way to deploy this either.